Completed Task 2393: Design System > MUI > Change theme and styling

rushigadekarWai opened the pull request:
>Counter Enhancement
>Dialog Enhancement
>File Uploader Enhancement
>Divider Enhancement
>Layout Enhancement

#Screenshots:
1.Counter:
<img width="1885" height="870" alt="image" src="https://github.com/user-attachments/assets/f9c96977-52c6-4716-b2be-85aa540531fe" />

2. Dialog: 
<img width="1876" height="887" alt="dialog" src="https://github.com/user-attachments/assets/dab3cc62-18f1-4d99-ad0b-4503a139bc67" />
3. File uploader: 
<img width="1888" height="935" alt="file uploder" src="https://github.com/user-attachments/assets/26107899-02e5-4e77-98ee-7340e30632a5" />
<img width="1897" height="928" alt="file uploder-2" src="https://github.com/user-attachments/assets/4e12f774-997b-47d8-b3fe-0d288698573b" />
<img width="1892" height="908" alt="file-uploader-3" src="https://github.com/user-attachments/assets/6e7a3302-cce0-4801-a890-e10bc4d6381b" />
<img width="1876" height="923" alt="file-uploader-4" src="https://github.com/user-attachments/assets/a7d9bfe2-41ac-4dd5-bb16-cb089e76c191" />

4.Layouts:
<img width="1896" height="932" alt="layout-1" src="https://github.com/user-attachments/assets/e8266293-b28d-4755-a2d6-8350b608c230" />
<img width="1920" height="1080" alt="layout-2" src="https://github.com/user-attachments/assets/a90566de-ec08-456f-a10f-810683a9cf5e" />
<img width="1872" height="891" alt="image" src="https://github.com/user-attachments/assets/fe870038-086f-4669-b95f-3c2dd0a614ae" />
<img width="1892" height="947" alt="image" src="https://github.com/user-attachments/assets/3dbfc5a3-0905-4d89-a963-2b95b289b0bd" />
<img width="1898" height="992" alt="image" src="https://github.com/user-attachments/assets/f3877d26-4390-4e79-87f8-2c4a10f6f98c" />
<img width="1872" height="957" alt="image" src="https://github.com/user-attachments/assets/df65ae3a-5a07-4d7f-93dd-b5a8e32f7012" />

rushigadekarWai committed:
Enhance RdsCounter and RdsDivider components with improved styling and functionality; update package dependencies

rushigadekarWai committed:
Enhance RdsFileUploader component with improved styling and functionality; add CSS for RdsDivider

rushigadekarWai committed:
Enhance RdsDialog and RdsFileUploader components with new features and improved styling; add support for title, dismiss button, and mandatory file validation

rushigadekarWai committed:
Enhance RdsDialog and RdsFileUploader components with new props and improved functionality; add support for title, close button, and mandatory validation

rushigadekarWai committed:
Enhance RdsFileUploader component with new styling options and improved functionality; add support for different drop area styles and hint text display

rushigadekarWai committed:
Enhance RdsCompLayout component with new display types and shadow support; add styling for Basic, Board, Boxify, Cardify, and Collage layouts

rushigadekarWai committed:
Refactor code structure for improved readability and maintainability

rushigadekarWai committed:
Merge branch 'new-solution' into users/rushikest/task-mui-doc

rushigadekarWai committed:
Refactor RDS Dialog component styles and update story with improved layout and content

rushigadekarWai committed:
Merge branch 'new-solution' into users/rushikest/task-mui-doc

rushigadekarWai committed:
Update RDS Dialog and Layout styles for improved consistency and responsiveness

rushigadekarWai committed:
feat: add RdsFileUploaderStandardView component for file upload functionality with drag-and-drop support

rushigadekarWai committed:
fix: update border-radius of .rds-comp-layout__paper for improved aesthetics

rushigadekarWai committed:
Merge branch 'new-solution' into users/rushikest/task-mui-doc

rushigadekarWai committed:
Merge branch 'new-solution' into users/rushikest/task-mui-doc

rushigadekarWai committed:
feat: add responsive styles for RDS File Uploader and update layout properties

supriyartambewaiin committed:
Merge branch 'new-solution' into users/rushikest/task-mui-doc

rushigadekarWai committed:
Refactor styles and components for consistency and improved theming

- Updated SCSS files for rds-counter, rds-dialog, and rds-file-uploader to utilize CSS variables for colors and backgrounds.
- Enhanced rds-file-uploader component to accept children as props for custom icons.
- Added new layout types 'Dashboard' and 'Relaxed' in rds-comp-layout with corresponding render logic.
- Improved layout styles to use CSS variables for spacing and shadows.
- Updated storybook stories for rds-comp-layout to include new layout types.

rushigadekarWai committed:
Merge branch 'users/rushikest/task-mui-doc' of https://github.com/Wai-Technologies/raaghu-react into users/rushikest/task-mui-doc

rushigadekarWai committed:
Implement code changes to enhance functionality and improve performance

supriyartambewaiin committed:
Merge branch 'new-solution' into users/rushikest/task-mui-doc

supriyartambewaiin commented on raaghu-elements/rds-file-uploader/rds-file-uploader.tsx:
```diff
@@ -143,143 +174,264 @@ const RdsFileUploader: React.FC<RdsFileUploaderProps> = ({
+                  aria-label="Remove file"
+                >
+                  {/* Cross SVG icon */}
+                  <svg width="18" height="18" viewBox="0 0 18 18" fill="none" xmlns="http://www.w3.org/2000/svg">
```
Don't add hard coded svg into code

supriyartambewaiin commented on raaghu-elements/rds-counter/rds-counter.scss:
All css should be come from global file

supriyartambewaiin committed:
Merge branch 'new-solution' into users/rushikest/task-mui-doc

supriyartambewaiin commented on raaghu-elements/rds-dialog/rds-dialog.scss:
css should be come from global file

supriyartambewaiin commented on raaghu-elements/rds-file-uploader/rds-file-uploader.scss:
all css should be come from global file

supriyartambewaiin commented on raaghu-elements/rds-file-uploader/rds-file-uploader.stories.tsx:
```diff
@@ -29,220 +29,252 @@ const meta: Meta<typeof RdsFileUploader> = {
+    showHint: true,
+     hintText: 'Maximum 5MB',
+},
+// render: (args) => {
```
remove commented code

supriyartambewaiin commented on raaghu-elements/rds-file-uploader/rds-file-uploader.stories.tsx:
```diff
@@ -29,220 +29,252 @@ const meta: Meta<typeof RdsFileUploader> = {
+    showHint: true,
+     hintText: 'Maximum 5MB',
+},
+// render: (args) => {
```
remove commented code

supriyartambewaiin commented on raaghu-elements/rds-file-uploader/rds-file-uploader.tsx:
```diff
@@ -143,143 +174,264 @@ const RdsFileUploader: React.FC<RdsFileUploaderProps> = ({
+            </Typography>
+            {files.map((fileWithProgress, index) => (
+              <Box
+                key={index}
```
don't use css in tsx file

supriyartambewaiin commented on raaghu-layouts/rds-comp-layout/rds-comp-layout.scss:
All css should be come from global file

supriyartambewaiin commented on raaghu-layouts/rds-comp-layout/rds-comp-layout.stories.tsx:
```diff
@@ -56,218 +66,260 @@ const SampleCard = ({ title }: { title: string }) => (
   </div>
 );
 
+// export const Default: Story = {
```
remove commented code

supriyartambewaiin commented on raaghu-layouts/rds-comp-layout/renderLayoutContent.tsx:
make sure if the code is lengthy create another file and call that file here. code should not be greater than 200 lines.

rushigadekarWai committed:
Refactor styles in RDS components to use CSS variables for spacing, font sizes, and colors

- Updated rds-counter.scss to replace fixed values with CSS variables for spacing, font sizes, and widths.
- Modified rds-dialog.scss to utilize CSS variables for padding, font sizes, and border radius.
- Enhanced rds-file-uploader.scss with CSS variables for margins, font sizes, and widths.
- Adjusted rds-file-uploader.tsx to implement CSS variables for styling, including icon colors and button dimensions.

rushigadekarWai committed:
Merge branch 'users/rushikest/task-mui-doc' of https://github.com/Wai-Technologies/raaghu-react into users/rushikest/task-mui-doc

rushigadekarWai committed:
Remove commented-out render functions from RdsFileUploader stories for cleaner code

rushigadekarWai committed:
Merge branch 'new-solution' into users/rushikest/task-mui-doc

rushigadekarWai committed:
Refactor RdsFileUploader component and add supporting components for improved file upload functionality
